src/goto-symex/field_sensitivity.h

```diff
@@ -31,14 +31,16 @@ class field_sensitivityt
   /// \param expr: an expression to be (recursively) transformed - this
   /// parameter is both input and output.
   /// \param write: set to true if the expression is to be used as an lvalue.
-  static void apply(const namespacet &ns, exprt &expr, bool write);
+  static void apply(
+    const namespacet &ns, exprt &expr, bool write, goto_symex_statet &state);
```

Comment:
The documentation should be updated for the new parameter :duck:

Reply:
Done.